setup.py

```diff
@@ -45,6 +45,7 @@
     'test': [
         # 4.2 <= pytest < 6.2 is slow collecting tests and times out on CI.
         # pytest < 7.2 has some different behavior that makes our CI fail
+        'packaging',
```

Comment:
Think it makes sense to keep `packaging` here if it is needed to run the tests (this doesn't affect runtime dependencies)